float64 x
float64 y
bool show_marker

---

bool marker_updated
